Procfile

```diff
@@ -1 +1 @@
-web: bin/www
+web: node bin/www
```

Comment:
Does this need to happen? I think it's an executable file (it's permissions are +x, and it has the she-bang `#!` at the beginning of the file), so the OS knows how to execute it without explicitly providing the interpreter.